colossalai/nn/optimizer/came.py

```diff
@@ -120,7 +121,7 @@ def step(self, closure=None):
                     exp_avg_sq.mul_(group["betas"][1]).add_(update, alpha=1.0 - group["betas"][1])
                     update = exp_avg_sq.rsqrt().mul_(grad)
 
-                update.div_((self._rms(update) / group["clip_threshold"]).clamp_(min=1.0))
+                # update.div_((self._rms(update) / group["clip_threshold"]).clamp_(min=1.0))
```

Comment:
Why

Reply:
> Why

open rms for both came and dist came in https://github.com/hpcaitech/ColossalAI/pull/5676/commits/74d4facc625e236d232b3ae845431537c9e08748；